components/decentralized_dns/content/decentralized_dns_interstitial_controller_client.cc

```diff
@@ -63,8 +63,7 @@ void DecentralizedDnsInterstitialControllerClient::SetResolveMethodAndReload(
   } else if (IsSnsTLD(request_url_.host_piece())) {
     pref_name = kSnsResolveMethod;
   } else {
-    NOTREACHED_IN_MIGRATION();
-    return;
+    NOTREACHED();
```

Comment:
where does `request_url_` comes from? It seems risky to have notreached (crash) if it's not really clear how it is formed

Reply:
https://github.com/brave/brave-core/blob/master/components/decentralized_dns/content/decentralized_dns_navigation_throttle.cc#L57-L68
https://github.com/brave/brave-core/blob/master/components/decentralized_dns/content/decentralized_dns_navigation_throttle.cc#L87-L96
So I think it's safe.